storage/streams/src/main/java/io/apicurio/registry/streams/utils/ArtifactKeyDeserializer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.streams.utils;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.apicurio.registry.storage.proto.Str;
+import org.apache.kafka.common.serialization.Deserializer;
+
+public class ArtifactKeyDeserializer implements Deserializer<Str.ArtifactKey> {
+
+	@Override
+	public Str.ArtifactKey deserialize(String topic, byte[] data) {
+		try {
+			return Str.ArtifactKey.parseFrom(data);
+		} catch (InvalidProtocolBufferException e) {
+			//FIXME remove
+			e.printStackTrace();
+		}
+		return null;
+	}
```

Comment:
Got some formatting issues here - using tabs instead of spaces.  IDE configured incorrectly?

Reply:
Looks like any *new* java classes have that issue...

Reply:
Sorry, class created **before** configuration change, let me fix them.

Reply:
Fixed